src/Core/Components/NavMenu/FluentNavBase.cs

```diff
@@ -98,9 +87,33 @@ public abstract class FluentNavBase : FluentComponentBase
     [Parameter]
     public bool ForceLoad { get; set; }
 
+    /// <summary>
+    /// Gets or sets the id of the custom toggle element
+    /// Defaults to navmenu-toggle
+    /// </summary>
+    [Parameter]
+    public string CustomToggleId { get; set; } = "navmenu-toggle";
+
+    [CascadingParameter]
+    public FluentNavMenu Owner { get; set; } = default!;
```

Comment:
Could we set these 2 `CascadingParameter` to `internal` ?

Reply:
Making them internal leas to errors in tests. Blazor also expects `[Parameter]` and `[CascadingParameter]` properties to be public

Reply:
I think `CascadingParameter` can be private or internal (ex in `FluentDataGridCell`).
And to access internal properties from unit tests, we've added
`[assembly: System.Runtime.CompilerServices.InternalsVisibleTo("Microsoft.FluentUI.AspNetCore.Components.Tests")]` 

Which error do you have in tests?

Reply:
When Owner is internal:  System.ArgumentException : The parameter selector 'p => p.Owner' does not resolve to a public property on the component 'Microsoft.FluentUI.AspNetCore.Components.FluentNavGroup' with a [Parameter] or [CascadingParameter]attribute. (Parameter 'parameterSelector') (in tests project)

Don't think it is worth the hassle to 'fight' this error and just leave them public as Blazor expects them to be